boards/sensebox/doc.txt

```diff
@@ -0,0 +1,56 @@
+/**
+ * @defgroup    boards_sensebox SenseBox
+ * @ingroup     boards
+ * @brief       Support for the SenseBox board.
+ *
+ * ### General information
+ *
+ * [SenseBox](https://www.sensebox.de/) is a do-it-yourself kit for stationary
+ * and mobile sensor stations.
+ *
+ * SenseBox board is based on the Atmel SAMD21G18A microcontroller. See
+ * @ref cpu_samd21.
+ *
+ *
+ * ### Flash the board
+ *
+ * 1. Put the board in bootloader mode by double tapping the reset button.<br/>
+ *    When the board is in bootloader mode, the user led (red) oscillates smoothly.
+ *
+ *
+ * 2. Use `BOARD=sensebox` with the `make` command.<br/>
+ *    Example with `hello-world` application:
+ * ```
+ *      make BOARD=sensebox -C examples/hello-world flash
+ * ```
+ *
+ * ### Accessing STDIO via UART
+ *
+ * To access the STDIO of RIOT, a FTDI to USB converted needs to be plugged to
+ * the UART/Serial 1 pins on the board.
+ *
+ * ### On I2C port
+ * I2C port is enabled by default. To disable it use ```I2C_DISABLE```. To
+ * re-enable use ```I2C_ENABLE```.
+ *
+ * ### On XBEE ports
+ * XBEE1 and XBEE2 ports are enabled by default. To disable any of them use
+ * ```XBEEn_DISABLE```. To re-enable use ```XBEEn_ENABLE```.
+ *
+ * On the board there is a reset circuit for this ports that delays the reset
+ * signal from the enable signal for about 300ms. Because of that a delay has
+ * been introduced in the ```board_init``` function, so that any SPI device
+ * connected to the ports is reset before use.
+ *
+ * #### SX127X radio
```

Comment:
Is there an sx127x on the board ? I would rather use a RN2483 module like [this one](https://shop.sodaq.com/lorabee-rn2483-order-now.html). It can be plugged directly on the XBee port and then use the rn2xx3 driver of RIOT.